code/game/gamemodes/clock_cult/clock_structures/traps/brass_skewer.dm

```diff
@@ -0,0 +1,88 @@
+//Non-servants standing over this will get spikes through the feet, immobilizing them until they're freed.
+/obj/structure/destructible/clockwork/trap/brass_skewer
+	name = "brass skewer"
+	desc = "A massive brass spike, cleverly concealed in the floor. You think you could prevent setting it off if you moved slowly."
+	clockwork_desc = "A barbaric but undeniably effective weapon: a spear through the chest. It immobilizes anyone unlucky enough to step on it and keeps them in place until they get help.."
+	icon_state = "brass_skewer"
+	break_message = "<span class='warning'>The skewer snaps in two!</span>"
+	max_integrity = 40
+	density = FALSE
+	can_buckle = TRUE
+	buckle_prevents_pull = TRUE
+	buckle_lying = FALSE
+	var/wiggle_wiggle
+
+/obj/structure/destructible/clockwork/trap/brass_skewer/Initialize()
+	. = ..()
+	START_PROCESSING(SSfastprocess, src)
+
+/obj/structure/destructible/clockwork/trap/brass_skewer/Destroy()
+	STOP_PROCESSING(SSfastprocess, src)
+	if(buckled_mobs && buckled_mobs.len)
+		var/mob/living/L = buckled_mobs[1]
+		L.Knockdown(100)
+		L.visible_message("<span class='warning'>[L] is maimed as the skewer shatters while still in their body!</span>")
+		L.adjustBruteLoss(15)
+	return ..()
+
+/obj/structure/destructible/clockwork/trap/brass_skewer/process()
+	if(density)
+		if(buckled_mobs.len)
+			var/mob/living/spitroast = buckled_mobs[1]
+			spitroast.adjustBruteLoss(0.1)
+			spitroast.dropItemToGround(spitroast.get_active_held_item()) //So they can't wield weapons, but can still try to free themselves
+
+/obj/structure/destructible/clockwork/trap/brass_skewer/bullet_act(obj/item/projectile/P)
+	if(buckled_mobs.len)
+		var/mob/living/L = buckled_mobs[1]
+		return L.bullet_act(P)
+	return ..()
+
+/obj/structure/destructible/clockwork/trap/brass_skewer/activate()
+	if(density)
+		return
+	var/mob/living/carbon/squirrel = locate() in get_turf(src)
+	if(squirrel)
+		squirrel.visible_message("<span class='boldwarning'>A massive brass spike erupts from the ground, impaling [squirrel]!</span>", \
+		"<span class='userdanger'>A massive brass spike rams through your chest, hoisting you into the air!</span>")
+		squirrel.emote("scream")
+		playsound(squirrel, 'sound/effects/splat.ogg', 50, TRUE)
+		playsound(squirrel, 'sound/misc/desceration-03.ogg', 50, TRUE)
+		squirrel.apply_damage(20, BRUTE, "chest")
+		squirrel.pixel_y = 8
```

Comment:
pretty sure that this isn't the proper way to handle buckling pixel offsets; see how rollerbeds do it